keps/sig-network/5495-deprecate-ipvs-mode-in-kube-proxy/README.md

```diff
@@ -0,0 +1,526 @@
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+At time of writing, kube-proxy has four supported backends (`iptables`, `ipvs`, `nftables`, and `winkernel`).
+Due to lack of maintainers of `ipvs`, sig-network has been encouraging users of `ipvs` to move to
+`nftables`, where they can, rather than continuing to maintain `ipvs`. (ie: [first example], [second example])
+
+`ipvs` was introduced in [KEP-265] to solve performance problems in large clusters.
+Since the GA of [KEP-3866], `nftables` provides an alternative to `ipvs` that fixes
+those performance issues on a modern supported kube-proxy backend.
+
+[First example]: https://github.com/kubernetes/kubernetes/issues/132689#issuecomment-3031585314
+[second example]: https://github.com/kubernetes/kubernetes/issues/132068#issuecomment-2945169346
+[KEP-265]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/265-ipvs-based-load-balancing/README.md
+[KEP-3866]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/3866-nftables-proxy/README.md
+
+### Goals
+
+- Update the kubernetes website with deprecation notices for `ipvs`
+- Print a warning when a user starts kube-proxy in `ipvs` mode
+
+### Non-Goals
+
+- Removal of `ipvs` - The specifics of removal of `ipvs` are being handled in [KEP-5343]
+
+[KEP-5343]: https://github.com/kubernetes/enhancements/pull/5344
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+
+As a user of `ipvs`, I need to know and understand that I'm running production clusters
+on a backend that isn't maintained, and need to be warned of that, so that I can make plans
+to move to `nftables`
```

Comment:
You can remove this. User stories are optional, and this one isn't saying anything non-obvious.